docker/web/public/getUsers.php

```diff
@@ -37,6 +37,13 @@
     </style>
 </head>
 <body>
+<?php
+$jackson = '';
+for ($x = 0; $x <= 4; $x++) {
+ $jackson .=  "The number is: $data[$x] <br>";
```

Comment:
This is an array, and won't print like that, as it's a an array and untemplated.
`"The number is: {$data[$x]}"`